src/TLuaInterpreter.cpp

```diff
@@ -6780,6 +6799,16 @@ int TLuaInterpreter::exists( lua_State * L )
             it1++;
         }
     }
+    //TODO: exists(keybind) doesn't currently work, figure out why
+    else if( type == "keybind")
+    {
+        QMap<QString, TKey *>::const_iterator it1 = pHost->getKeyUnit()->mLookupTable.find( name );
+        while( it1 != pHost->getKeyUnit()->mLookupTable.end() && it1.key() == name )
```

Comment:
As it turns out, exists() doesn't work for keybinds right now. I thought I'd sorted that out, but just tested it, and it fails. So your suggestions may solve the problem I'm running into also.